docs/en/dev/adapterdev.md

```diff
@@ -316,13 +318,37 @@ Short structure of package.json:
 ```
 - All fields are mandatory. devDependencies should be inside too to enable the grunt tasks.
 
+### Versioning
+
+It is required to use [Semantic Versioning](https://semver.org/) (SemVer) for your adapters.
+
+#### Syntax
+
+Version 1.2.3 =>
+
+| **Major** | **Minor** | **Patch** |
+|-----------|-----------|-----------|
+| 1         | 2         | 3         |
+
+#### Explanation
+
+* Major: Changes which lead to an incompatibility with previous versions
+* Minor: Added or extended functionality  (backwards compatible)
+* Patch: Bug fixes (backwards compatible)
+
+
 ### Deploying
-It is suggested to have the code on github. After the code is stable and lets to install adapter you can share you adapter to other user by asking them to install adapter as follow:
 
+It is recommended to have the adapter code on github. Once your code is stable and lets you install the adapter in your environment successfully, you can share your adapter to other users by one of the following ways:
+
+1. Installing within the ioBroker admin interface: Refer users to section *(5.) Install adapter from your own URL* of [ioBroker Admin documentation](https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/admin/adapter.md). 
+
+2. Alternatively, ask users to install the adapter through the command line: 
```

Comment:
If we add that there then also the required "iobroker upload name" needs to be added because without it will not work!

Way 1 ist the preferred one !!

Reply:
Probably better to entirely remove second option (command line). Detailed description is anyway [here](https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/config/cli.md#npm-install-iobrokeradaptername) and [here](https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/config/cli.md#iobroker-upload-adaptername).

Reply:
We can leave in but should state very loud that #1 is the preferred way. Maybe also add how to install from a branch via that

Reply:
Added accordingly. Further additions can be added in additional documentation improvements, there are many more parts to improve and extend, but let's start with at least something :-)